webserver/webserver/src/main/java/io/helidon/webserver/WebServer.java

```diff
@@ -751,6 +751,17 @@ public Builder workersCount(int workers) {
             return this;
         }
 
+        /**
+         * Configure whether to enable content negotiation for compression.
```

Comment:
This should maybe say "for default socket", as this is socket specific, if I understood this PR correctly